machine_manager/README.rst

```diff
@@ -0,0 +1,27 @@
+Machine Manager
+===============
+
+This module is a vertical for Machinery management.
```

Comment:
description could certainly be improved: machine manager is a module to manage machines... I'm not enlightened...